writeups/vn.txt

```diff
@@ -0,0 +1,133 @@
+to answer all of them.
+
+## Document History
+
+1. Does the working group (WG) consensus represent the strong concurrence of a
+   few individuals, with others being silent, or did it reach broad agreement?
+Broad agreement. This document was widely discussed on github, the mailing list, and at multiple IETF meetings.
+
+2. Was there controversy about particular points, or were there decisions where
+   the consensus was particularly rough?
+Given the nature of the work (versions and moving between versions) there were some philosophical questions that had to be ironed out, but ultimately the consensus was not unusually rough.
+
+3. Has anyone threatened an appeal or otherwise indicated extreme discontent? If
+   so, please summarize the areas of conflict in separate email messages to the
+   responsible Area Director. (It should be in a separate email because this
+   questionnaire is publicly available.)
+No.
+
+4. For protocol documents, are there existing implementations of the contents of
+   the document? Have a significant number of potential implementers indicated
+   plans to implement? Are any existing implementations reported somewhere,
+   either in the document itself (as [RFC 7942][3] recommends) or elsewhere
+   (where)?
+There are multiple interoperating implementations which have been reported and tested through the typical form of social testing on the QUIC dev slack workspace. This includes major implementers who intend to use it in production.
+
+### Additional Reviews
+
+5. Do the contents of this document closely interact with technologies in other
+   IETF working groups or external organizations, and would it therefore benefit
+   from their review? Have those reviews occurred? If yes, describe which
+   reviews took place.
+No.
+
+6. Describe how the document meets any required formal expert review criteria,
+   such as the MIB Doctor, YANG Doctor, media type, and URI type reviews.
+None.
+
+7. If the document contains a YANG module, has the final version of the module
+   been checked with any of the [recommended validation tools][4] for syntax and
+   formatting validation? If there are any resulting errors or warnings, what is
+   the justification for not fixing them at this time? Does the YANG module
+   comply with the Network Management Datastore Architecture (NMDA) as specified
+   in [RFC 8342][5]?
+N/A
+
+8. Describe reviews and automated checks performed to validate sections of the
+   final version of the document written in a formal language, such as XML code,
+   BNF rules, MIB definitions, CBOR's CDDL, etc.
+Normal linting.
+
+### Document Shepherd Checks
+
+9. Based on the shepherd's review of the document, is it their opinion that this
+   document is needed, clearly written, complete, correctly designed, and ready
+   to be handed off to the responsible Area Director?
+I believe these documents are complete and clear and ready for AD review.
```

Comment:
Maybe also mention "needed" and "correctly designed"?

Reply:
yeah agree here, the answer is an easy direct one - omission here sticks out